pkg/util/docker/event_pull.go

```diff
@@ -78,10 +78,14 @@ func (d *DockerUtil) processContainerEvent(ctx context.Context, msg events.Messa
 	}
 
 	action := msg.Action
-
-	// Fix the "exec_start: /bin/sh -c true" case
-	if strings.Contains(action, ":") {
-		action = strings.SplitN(action, ":", 2)[0]
+	// Some actions are prefixed then followed `: <string>`.
+	// Extract the prefix from the action here
+	// Example: "exec_start: /bin/sh -c true" case
+	for _, prefix := range actionPrefixes {
+		if strings.HasPrefix(string(action), string(prefix)) {
+			action = prefix
```

Comment:
What about:
```
ActionHealthStatusRunning   Action = "health_status: running"
ActionHealthStatusHealthy   Action = "health_status: healthy"
ActionHealthStatusUnhealthy Action = "health_status: unhealthy"
```

Which you include in he

```
var actionPrefixes = []events.Action{
	events.ActionExecDie,
	events.ActionExecStart,
	events.ActionExecDetach,
	events.ActionExecCreate,
	events.ActionHealthStatusRunning,
	events.ActionHealthStatusHealthy,
	events.ActionHealthStatusUnhealthy,
}
```
Should not that be:

```
var actionPrefixes = []events.Action{
	events.ActionExecDie,
	events.ActionExecStart,
	events.ActionExecDetach,
	events.ActionExecCreate,
	events.ActionHealthStatus,
}
```
since

```
ActionHealthStatusRunning   Action = "health_status: running"
ActionHealthStatusHealthy   Action = "health_status: healthy"
ActionHealthStatusUnhealthy Action = "health_status: unhealthy"
```

not only include the prefix but the actual : something ?